src/ui/TypingIndicatorMessage/index.tsx

```diff
@@ -1,48 +1,36 @@
-import React, {ReactElement, useContext, useRef} from 'react';
+import React, {ReactElement, useContext} from 'react';
 import './index.scss';
-import {getClassName} from '../../utils';
-import Avatar from '../Avatar';
 import {Member} from '@sendbird/chat/groupChannel';
 // @ts-ignore
 import typingIndicatorLogo from '../../gifs/message-typing-indicator.gif';
 import {LocalizationContext} from '../../lib/LocalizationContext';
-
-const AVATAR_SIZE = '28px';
+import MessageProfile from '../MessageProfile';
+import MessageHeader from '../MessageHeader';
 
 export interface TypingIndicatorMessageProps {
-  className?: string | Array<string>;
   typingMember: Member;
 }
 
 export default function TypingIndicatorMessage({
-  className,
   typingMember,
 }: TypingIndicatorMessageProps): ReactElement {
-  const avatarRef = useRef(null);
   const { stringSet } = useContext(LocalizationContext);
 
   return (
-    <div className={getClassName([
-      className,
-      'sendbird-typing-indicator-message',
-    ])}>
-      <div className='sendbird-typing-indicator-message__sender-profile-container'>
-        {
-          <Avatar
-            src={typingMember.profileUrl}
-            ref={avatarRef}
-            width={AVATAR_SIZE}
-            height={AVATAR_SIZE}
-          />
-        }
+    <div className='sendbird-message-content incoming'>
+      <MessageProfile member={typingMember}/>
+      <div className='sendbird-message-content-middle'>
+        <div>
+          <MessageHeader text={`${typingMember.nickname} is typing...`}/>
```

Comment:
The intention is somewhat confusing.

The reason for separating the public and private props interfaces seems to be based on their intended usage within the internals, rather than for the purpose of exposing them to customers.
It would be good to clarify this aspect.